From 1a869a65719f14f702dff492a6f90b135b9d0646 Mon Sep 17 00:00:00 2001
From: Georgi Valkov <gvalkov@gmail.com>
Date: Fri, 19 Jul 2024 13:24:27 +0300
Subject: [PATCH 083/125] gitignore: add .DS_Store

macOS creates .DS_Store files all over the place while browsing
directories. Add it to the list of ignored files.

Signed-off-by: Georgi Valkov <gvalkov@gmail.com>

Reviewed-by: Tomas Mraz <tomas@openssl.org>
Reviewed-by: Todd Short <todd.short@me.com>
(Merged from https://github.com/openssl/openssl/pull/24942)

(cherry picked from commit 10c36d2f8d81a6f2b9a75f914fe094300835ba01)
(cherry picked from commit 97b2aa49e504e49a8862b89c65eb54e143395f1d)
---
 .gitignore | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/.gitignore b/.gitignore
index 98184721f2..d4ee22c47c 100644
--- a/.gitignore
+++ b/.gitignore
@@ -147,6 +147,9 @@ providers/common/include/prov/der_sm2.h
 /apps/progs.c
 /apps/progs.h
 
+# macOS
+.DS_Store
+
 # Windows (legacy)
 /tmp32
 /tmp32.dbg
-- 
2.39.5 (Apple Git-154)

